config/routes.rb

```diff
@@ -248,6 +248,7 @@
     AcuantSdkController::ACUANT_SDK_STATIC_FILES.each do |acuant_sdk_file|
       get "/verify/doc_auth/#{acuant_sdk_file}" => 'acuant_sdk#show'
       get "/verify/capture_doc/#{acuant_sdk_file}" => 'acuant_sdk#show'
+      get "/verify/capture-doc/#{acuant_sdk_file}" => 'acuant_sdk#show'
```

Comment:
more dashes....ugh maybe I should have just made a rack filter that upcases the URL instead of trying to remember where to dash?

Reply:
Are we explicitly linking here? Should we define `_dashes` name for the URL (like `capture_doc_step_dashes`) like we did for the others? Is there anywhere we can test for this?

Reply:
We are not explicitly linking here. The reason this is here is the SDK expects these to be available at a path relative to the current path.

I could add a request spec here, but I can't think of a really good way to robustly test it right now.

Reply:
Request spec doesn't super make sense, probably fine to just merge it